tools/shuttle/src/store/root.rs

```diff
@@ -41,6 +43,23 @@ impl RootStorage {
             None => None,
         };
 
+        #[cfg(target_arch = "wasm32")]
+        let root_cid = match path.as_ref() {
+            Some(path) => path.to_str().and_then(|key| {
+                use gloo::storage::{LocalStorage, Storage};
+                let local_storage = LocalStorage::raw();
+                local_storage.get(key).ok().and_then(|key| {
+                    key.and_then(|key|{
+                        LocalStorage::get(key).ok().and_then(|bytes: Vec<u8> |{
+                            let cid_str = String::from_utf8_lossy(&bytes).to_string();
+                            cid_str.parse().ok()
+                        })
+                    })
+                })
+            }),
+            None => None,
+        };
```

Comment:
This wouldnt be needed since the root store is apart of the server-side code so we could use `#[cfg(not(target_arch = "wasm32"))]` on https://github.com/Satellite-im/Warp/blob/b574da4ab10cfd7bf38ec0e2b499232a2e8f8eff/tools/shuttle/src/lib.rs#L16-L18 which should allow shuttle to compile.

Reply:
done